articles/tech-lead-needed.md

```diff
@@ -0,0 +1,98 @@
+
+* Furtherwise, a good tech lead will have a sixth sense for technical risks and unexpected effort. For example, the software upgrade mechanisms and the configuration management for distributed systems are usually underestimated, and a tech lead will keep that in mind when estimating a project's cost.
+
+In all modesty, I could go on endlessly. For example, we didn't even begin to investigate interpersonal matters. However, since this is supposed to be an article rather than a textbook, we should now head on to the second key factor:
+
+## Preparing a Tech Lead
+
+As pointed out above, it is relatively easy to teach formal knowledge (such as linear algebra or physical laws, for example), but it is close to impossible to teach experience. That is why reading one, three or even a thousand books on sailing, however wise and excellent they may be, wouldn't magically turn you into a good seafarer; you would still need a good bit of practice. Similarly, you cannot fully prepare a software engineer for a tech-lead role by making them read a book or by sending them, say, to an ISAQB course -- no matter how excellent the trainers are.
+
+How, then, can we prepare them instead? The answer is quite simple: We need to take them by the hand and lead them through incremental challenges in which they can experience the new responsibilities bit by bit.
+
+In practice, this can look as follows:
+
+* If you feel that some senior developer has the potential to become a true tech lead, give them the responsibility for a subsystem in a large project (or the entire system in a small project) for starters.
+
+* Provide mentoring or coaching. The larger the gap between the candidate's previous responsibilities and the new challenge, the more intense the mentoring/coaching needs to be.
+
+* Encourage new tech leads to network with established tech leads as well as other newcomers. Give them some room to reflect on their often overwhelming impressions in a circle of trustful peers.
+
+* Do not officially advance an engineer to a tech lead position before they had a chance of proving themselves in that role (because if you do so, this will seem noncredible to the team and consequently undermine the newcomer's standing).
+
+## Final Remarks
+ 
+Leadership is a very complex subject, and hence, it is a bold undertaking to try and describe how it works in a single article. Moreover, I need to point out again that this article was written from my perspective, which certainly is quite specific. Most certainly, my view does not represent all projects ever conducted by the Zühlke Technology Group. Nonetheless, I hope that my perceptions contain something that you can put to use, and that I managed to resolve the mystery of the tech lead question by at least a tiny bit.
```

Comment:
As this (starting from the second sentence) applies to most of the articles, I think about adding a disclaimer to the collection. I'll give you an update.